db/seeds.rb

```diff
@@ -19,15 +19,15 @@
 
 #Friends
 30.times do
-  Friend.create(first_name: FFaker::Name.first_name, 
-    last_name: FFaker::Name.last_name, 
-    a_number: rand.to_s[2..10], 
+  Friend.create(first_name: FFaker::Name.first_name,
```

Comment:
It's generally best practice to keep any files that have unintentional space changes out of the branch.  I will typically check out an individual file from origin master to overwrite files I don't want changes to.  I use this 'git checkout origin/source_branch path/to/file'.  Don't worry about making the change for this PR, since you are on the master branch of a forked repo, it's going to be really tricky to do this, so leave it as is, just a note for the future!

Reply:
That's really good advice! I agree might be hard to implement here at this stage, but I can see how that would helpful and will definitely keep it in mind.